pkg/target/kusttarget_configplugin.go

```diff
@@ -78,6 +79,17 @@ func (kt *KustTarget) configureBuiltinTransformers(
 		}
 		result = append(result, r...)
 	}
+
+	simpleConfigurators := []simpleTransformerConfigurator{
+		kt.configureBuiltinReplicaCountTransformer,
+	}
+	for _, f := range simpleConfigurators {
```

Comment:
I opted to add another loop to go over `simpleTransformerConfigurator`s since the signature is different. I could have also implemented it with one loop and type swtiching, but I assume this approach is preferable?